public/app/features/alerting/unified/components/DynamicTable.tsx

```diff
@@ -197,7 +197,6 @@ const getStyles = <T extends unknown>(
     `,
     bodyCell: css`
       overflow: hidden;
-      word-break: break-all;
```

Comment:
I had to remove this becuase "break-all" would create: 

```
Alerting (Noda
ta)
```

whereas the default "normal" setting does

```
Alerting
(Nodata)
```

This line was added in a change specific to the alerting instances table, so I feel ok removing it for an alerting-instances-table improvement. I tested this on a mobile viewport and the table overflow was indistinguishable whatever the word-break setting was. https://github.com/grafana/grafana/commit/e5f6ab7fd2cb34fe21b88c6c53e86e1dd375024d